packages/venia-ui/lib/components/RichContent/richContent.js

```diff
@@ -21,20 +18,26 @@ const toHTML = str => ({ __html: str });
  * @returns {React.Element} A React component that renders Heading with optional styling properties.
  */
 const RichContent = props => {
-    const { html } = props;
     const classes = mergeClasses(defaultClasses, props.classes);
-
-    if (detectPageBuilder(html)) {
-        return (
-            <div className={classes.root}>
-                <PageBuilder masterFormat={html} />
-            </div>
+    const newProps = {
```

Comment:
Consider avoiding `new` in vars. 👍 

```suggestion
    const nextProps = {
```

Reply:
Thanks. Renamed to be more specific.